io/disk/tiobench.py.data/README.txt

```diff
@@ -0,0 +1,13 @@
+This program runs  a multi-threaded I/O benchmark test to measure file system performance in four basic operations: 
```

Comment:
Do you insist on having this extra file here? I'd prefer putting this info as header docstring of the test itself, which is the first thing people see when opening the test. Additionally there are efforts in providing the test's docstring on dry-runs/inquire (not yet settled, but the idea is to have reliable source of information for each test without the need to opening the sources).

Anyway it's not a strong demand, perhaps a question on how are you using this info. Our QA uses the docstring to learn about the test.